.circleci/config.yml

```diff
@@ -2314,6 +2326,29 @@ jobs:
       - run-eet-suites:
           dsl-args: "CannotDeleteSystemEntitiesSuite"
 
+  run-schedule-scenarios:
+    parameters:
+      runner:
+        type: executor
+        default: ci-test-executor
+      workflow-name:
+        type: string
+        default: ""
+    executor:
+      name: ci-test-executor
+      workflow-name:  << parameters.workflow-name >>
+    steps:
+      - run-eet-suites:
+          dsl-args: "ScheduleDeleteSpecs"
```

Comment:
Should we add these tests to `continuous-integration-gcp` as well ?

Reply:
That would be good 💯